Source/themes.cpp

```diff
@@ -348,6 +350,8 @@ static bool SpecialThemeFit(int i, theme_id t)
 			treasureFlag = false;
 		}
 		break;
+	default:
+		break;

```

Comment:
```suggestion
		app_fatal("Invalid theme.");
```

Reply:
Some types are intentionally left out of this swtich